utils.py

```diff
@@ -175,3 +175,12 @@ def validate_cookies_del(flows: list[dict]) -> None:
                 "cookie_mask is set, cookie should be set too"
                 f"when deleting flows. flow: {flow}"
             )
+
+
+def flows_to_log_info(message: str, flow_dict: dict[str, list]) -> None:
+    """Log flows, maximun flows in a log is 200"""
+    maximun = 200
+    i, j = 0, maximun
+    while flow_dict["flows"][i:j]:
+        log.info(f"{message} {flow_dict['flows'][i:j]}")
```

Comment:
Could you include a `f"flows indexes: [{i}:{j}]"` or something similar in this log entry here? just so when a slice iteration happens it's also easier to spot and understand that it's not a different request/event.

Reply:
Message updated to show indexes from `i` to the last index available or `j`.

Reply:
Excellent. Great log formatting.